packages/ember-metal/lib/meta.js

```diff
@@ -46,6 +47,24 @@ let members = {
   tag: ownCustomObject
 };
 
+// This is less than ideal, but `runInDebug` is
+// not populated by the time we need it here. So
+// we create a function that has the same signature
+// as the real `runInDebug` so that it is stripped
+// from the production build. Unfornately, this
+// function will remain in production build.
+let runInDebug = (fn) => fn();
```

Comment:
@krisselden - Yes, I recall discussing this with you. 

I think we decided the right path was to just ensure ember-debug was always present and swap the underlying implementation based on debug vs production build. I will try to work on this today...